lib/cocoapods/installer/analyzer.rb

```diff
@@ -35,7 +35,7 @@ class Analyzer
       #
       attr_reader :podfile
 
-      # @return [Lockfile] The Lockfile, if available, that stores the information about the Pods previously installed.
+      # @return [Lockfile, nil] The Lockfile, if available, that stores the information about the Pods previously installed.
```

Comment:
this clears up some warnings in my IDE (RubyMine) because its trying to infer what one can pass here or what can be returned.